mage_ai/data_preparation/models/constants.py

```diff
@@ -19,6 +19,7 @@
 DATAFRAME_SAMPLE_COUNT_PREVIEW = 10
 DATAFRAME_SAMPLE_COUNT = 1000
 DATAFRAME_SAMPLE_MAX_COLUMNS = 1000
+PRINT_OUTPUT_MAX_LINES = 100
```

Comment:
If `100` lines for the print output doesn't seem like enough, we can increase it.

Reply:
Can we set it as an environment variable? It's only needed by certain envrionments.

Reply:
@wangxiaoyou1993 Does a [default](https://github.com/mage-ai/mage-ai/pull/2659/files#diff-ca01b0d0977f659f1fb140c7f3cfa840dea282ba4bceec1122326ff04cf54286R23) of `1000` seem reasonable or something higher?

Reply:
1000 sounds reasonable